src/components/TasksTable/index.jsx

```diff
@@ -143,6 +149,18 @@ export default class TasksTable extends Component {
     this.setQuery(newQuery);
   };
 
+  handleRandomButtonClick = () => {
+    const { items } = this.props;
+    const unassignedItems = items.filter(item => item.assignee === '-');
```

Comment:
We should create a utility function for unassigned and assigned filter functions since we are duplicating these functions now. Create `utils/assignment_filters`:

```jsx
export const unassigned = ({ assignee }) => assignee === '-';
export const assigned = ({ assignee }) => assignee !== '-';

```